tests/platform_tests/test_engine.py

```diff
@@ -83,15 +87,16 @@ class TestDialogCreation(TestEngineBase):
     Tests how engines construct and show dialogs.
     """
 
-    @skip_if_pyside_missing
     def setUp(self):
         """
         We need a QApplication to run these tests.
         """
         super(TestDialogCreation, self).setUp()
 
-        if sgtk.platform.qt.QtGui.QApplication.instance() is None:
-            sgtk.platform.qt.QtGui.QApplication([])
+        from tank.authentication.ui.qt_abstraction import QtGui
```

Comment:
This test was failing left and right if executed by ifself because sgtk.platform.qt would not be initialized yet, so we're fixing it by using the qt importer.

Reply:
Maybe it would be good to add a comment explaining why in the test.